### PR TITLE
Release 1.8.1

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
Could you please take a look when you have a chance, @rmax?

**Comment by rmax** [comment]
Looks good to me, thanks.

**Comment by jakirkham** [comment]
Thanks. :)

File: recipe/build.sh [hunk 1]
```diff
@@ -4,6 +4,12 @@ set -ex
 export CFLAGS="${CFLAGS} -O3 -fPIC"
 export LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib"
 
+# Need librt on Linux for time operations.
+if [ "$(uname)" == "Linux"  ]
+then
+  export LDFLAGS="${LDFLAGS} -lrt"
+fi
```
Comment (jakirkham) [inline recipe/build.sh:11]: Note: This is a workaround for issue ( https://github.com/lz4/lz4/issues/450 ). FTR we can drop this once commit ( https://github.com/lz4/lz4/commit/ad12efa4e80855eb6267f5d3a7c50669e0565919 ) lands in a release.

**Pull request merged**